docs/csharp/whats-new/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: What's New in C# | C# Guide
+description: How is the C# language evolving
+keywords: C#, Latest Features, What's New, Roslyn
+author: BillWagner
+ms.author: wiwagn
+ms.date: 10/14/2016
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
+---
+
+# What's new in C# #
+
+
+* [C# 7](csharp-7.md):
+    - This page describes the latest features in the C# language. This covers C# 7, currently available in [Visual Studio 2017](https://www.visualstudio.com/vs/visual-studio-2017-rc/).
```

Comment:
Thanks for catching this Maira. I changed the link to https://www.visualstudio.com/vs/whatsnew/ (a really nice looking page that has the download link front and center).